### PR TITLE
Removed unimplemented fields to clean up tile UX. 

### PR DESCRIPTION
**Pull request by nstewart**
Removed fields associated with the currently broken SSL mode to simplify the UX while we gather feedback and finish testing.

Additionally, this update removes hard-coded host defaults and updates descriptive metadata.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -37,7 +37,6 @@ func main() {
 	log.Info("Starting CF CockroachDB broker")
 
 	InitServicesAndPlans()
-	defer CleanupPlans()
```
Comment (BramGruneir) [inline main.go:40]: Why did you remove this line?  What does it do?

File: manifest.yml [hunk 2]
```diff
@@ -13,17 +13,17 @@ applications:
...
           "metadata": {
             "displayName": "CockroachDB",
-            "longDescription": "A cloud-native SQL database for building global, scalable cloud services that survive disasters.",
+            "longDescription": "CockroachDB service broker for creating cloud-native SQL databases.",
             "documentationUrl": "https://www.cockroachlabs.com/docs/",
             "supportUrl": "https://www.cockroachlabs.com/community/",
             "imageUrl": "https://www.cockroachlabs.com/images/CockroachLabs_Logo_Mark-lightbackground.svg"
           },
-          "tags": ["cockroachdb", "relational"]
+          "tags": ["cockroachdb", "database", "SQL", "cloud", "kubernetes"]
```
Comment (BramGruneir) [inline manifest.yml]: Not sure if Kubernetes fits here.

File: tile.yml [hunk 3]
```diff
@@ -25,17 +25,17 @@ packages:
...
             "metadata": {
               "displayName": "CockroachDB",
-              "longDescription": "A cloud-native SQL database for building global, scalable cloud services that survive disasters.",
+              "longDescription": "CockroachDB service broker for creating cloud-native SQL databases.",
               "documentationUrl": "https://www.cockroachlabs.com/docs/",
               "supportUrl": "https://www.cockroachlabs.com/community/",
               "imageUrl": "https://www.cockroachlabs.com/images/CockroachLabs_Logo_Mark-lightbackground.svg"
             },
-            "tags": ["cockroachdb", "relational"]
+            "tags": ["cockroachdb", "database", "SQL", "cloud", "kubernetes"]
```
Comment (BramGruneir) [inline tile.yml]: same comment about kubernetes here.

File: manifest.yml (hunk 2 above)
Comment (nstewart) [reply manifest.yml]: Ok -- I'll remove

File: tile.yml (hunk 3 above)
Comment (nstewart) [reply tile.yml]: I'll remove

File: main.go (hunk 1 above)
Comment (nstewart) [reply main.go:40]: That removed the temporary files for holding the CA, cert, and key files which we no longer create

**Review by BramGruneir — APPROVED** [review]
LGTM!
